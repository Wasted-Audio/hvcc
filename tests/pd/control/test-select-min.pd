#N canvas 218 370 817 539 10;
#X msg 533 171 1;
#X msg 501 171 54;
#X msg 470 171 23;
#X msg 434 172 234;
#X msg 73 187 6;
#X msg 36 187 234;
#X msg 598 185 symbol cort;
#X msg 637 212 symbol zack;
#X msg 663 237 symbol bill;
#X obj 36 26 loadbang;
#X obj 36 162 t b b;
#X obj 40 488 print A;
#X obj 236 486 print B;
#X obj 210 156 t b b;
#X msg 210 206 symbol cort;
#X obj 210 241 sel cort;
#X obj 434 139 t b b b b;
#X obj 369 481 print C;
#X obj 515 479 print D;
#X obj 597 275 select cort zack;
#X obj 434 210 sel 1 54 23;
#X obj 36 48 t b b b b;
#X obj 598 131 t b b b;
#X obj 36 247 select 6;
#X msg 247 181 symbol zack;
#X connect 0 0 20 0;
#X connect 1 0 20 0;
#X connect 2 0 20 0;
#X connect 3 0 20 0;
#X connect 4 0 23 0;
#X connect 5 0 23 0;
#X connect 6 0 19 0;
#X connect 7 0 19 0;
#X connect 8 0 19 0;
#X connect 9 0 21 0;
#X connect 10 0 5 0;
#X connect 10 1 4 0;
#X connect 13 0 14 0;
#X connect 13 1 24 0;
#X connect 14 0 15 0;
#X connect 15 0 11 0;
#X connect 15 1 12 0;
#X connect 16 0 3 0;
#X connect 16 1 2 0;
#X connect 16 2 1 0;
#X connect 16 3 0 0;
#X connect 19 0 11 0;
#X connect 19 1 12 0;
#X connect 19 2 17 0;
#X connect 20 0 11 0;
#X connect 20 1 12 0;
#X connect 20 2 17 0;
#X connect 20 3 18 0;
#X connect 21 0 10 0;
#X connect 21 1 13 0;
#X connect 21 2 16 0;
#X connect 21 3 22 0;
#X connect 22 0 6 0;
#X connect 22 1 7 0;
#X connect 22 2 8 0;
#X connect 23 0 11 0;
#X connect 23 1 12 0;
#X connect 24 0 15 0;
