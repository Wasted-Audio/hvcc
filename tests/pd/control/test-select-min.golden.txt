[@ 0.000] C: symbol bill
[@ 0.000] B: bang
[@ 0.000] A: bang
[@ 0.000] A: bang
[@ 0.000] B: bang
[@ 0.000] C: bang
[@ 0.000] D: 234
[@ 0.000] B: symbol zack
[@ 0.000] A: bang
[@ 0.000] A: bang
[@ 0.000] B: 234